index.html

```diff
@@ -10,6 +10,14 @@
 
 <body>
   <header>
+    <div class="darkheader">
```

Comment:
- As requested by the previous reviewer your dark top bar is still missing the socail media icon which are visible from the design template provided. Kindly add them to your project design :+1: 
- design template
![sm-ic](https://github.com/mj-e-boc/Amapiano-Festival/assets/89055502/e4504f21-353b-4611-8f24-f3ef0b5e3ece)
- your design
![sm](https://github.com/mj-e-boc/Amapiano-Festival/assets/89055502/a1c60b1e-8ef9-4ac7-b532-11add3d35d29)